.gitignore

```diff
@@ -7,23 +7,20 @@
 *.wast.hpp
 *.dot
 *.abi.hpp
-*.cmake
```

Comment:
I don't quite understand the removal of this one...  won't it cause all the various cmake files in the build dir be flagged? Like `CPackSourceConfig.cmake` and `CTestTestfile.cmake` etc etc?

Reply:
We have this rule in `.gitignore`:

`[Bb]uild*/*`

Reply:
ah, true. but if the gitignore only supports build directories named `build*` or `cmake-build-{release,debug}`, then quite a bit of other cruft can likely be removed too

Reply:
I'm all for cleaning it up further. I think we should assume that the build dirs are already excluded, as this is normally the case